declarations_site/catalog/elastic_models.py

```diff
@@ -99,80 +94,79 @@ def get_family_members(self):
                     yield member["family_name"]
 
 
-
-class Declaration(AbstractDeclaration):
+class Declaration(DocType, RelatedDeclarationsMixin):
```

Comment:
It contains textual information, extracted from the declaration document and used for full text search option.

NACP declarations has similar field.